exec/java-exec/src/main/java/org/apache/drill/metastore/expr/StatisticsConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.metastore.expr;
+
+public interface StatisticsConstants {
```

Comment:
Similar to the prior comment about 'Exact' statistics, let's start differentiating between these and the estimated statistics.  @gparai  I believe you have a similar set of constants for the estimated stats such as NDV, NonNullRowCount etc.    It would be better to be explicit by prefixing 'ExactStatistics' vs. 'EstimatedStatistics' .  Otherwise there's going to be confusion.

Reply:
Agree, thanks, will rename it to `ExactStatisticsConstants`.